build-tests/api-extractor-scenarios/etc/test-outputs/ambientNameConflict/api-extractor-scenarios.api.ts

```diff
@@ -1,3 +1,5 @@
+// Warning: (ae-forgotten-export) The symbol "Promise" needs to be exported by the entry point index.d.ts
```

Comment:
Are these issues validated in any non-manual way? Maybe these files should be Jest snapshots instead.

Reply:
I know changes here create diffs, but I'm not sure it's obvious that these diffs are test outputs.

Maybe they are...

Reply:
We really need a build script that will break a production build if the contents of these file has changed, but the person forgot to commit those changes to Git.  Right now I simply have to remember to do it when I work on API Extractor.

Out of scope for this PR, though.